app/styles/app.scss

```diff
@@ -142,11 +166,14 @@ h2 {
 }
 
 .main_title {
-  background-color: $blue;
-  color:$white;
+  background-color: white;
+  color: black;
+  -webkit-box-shadow: inset 0 0 100px rgba(0,0,0,.5);
+
+  box-shadow: inset 0 0 100px rgba(0,0,0,.5);
 
   h1{
-    color: $red;
+    color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.